core/domain/cron_services.py

```diff
@@ -30,12 +30,11 @@
 
 (base_models, job_models, user_models) = models.Registry.import_models([
     models.NAMES.base_model, models.NAMES.job, models.NAMES.user])
+datastore_services = models.Registry.import_datastore_services()
 
 PERIOD_TO_HARD_DELETE_MODELS_MARKED_AS_DELETED = datetime.timedelta(weeks=8)
 PERIOD_TO_MARK_MODELS_AS_DELETED = datetime.timedelta(weeks=4)
-MODEL_CLASSES_TO_MARK_AS_DELETED = [
-    user_models.UserQueryModel
-]
+MODEL_CLASSES_TO_MARK_AS_DELETED = [user_models.UserQueryModel]
```

Comment:
Maybe better to use a tuple here (non-modifiable).

Reply:
Done.